interface/therapy_groups/therapy_groups_views/groupDetailsParticipants.php

```diff
@@ -0,0 +1,279 @@
+                                                <?php endif; ?>
+                                            </tr>
+                                            </thead>
+                                            <tbody>
+                                            <?php foreach ($participants as $i => $participant) : ?>
+                                                <tr>
+                                                    <td>
+                                                        <input type="hidden" name="pid[]" value="<?php echo htmlspecialchars($participant['pid'],ENT_QUOTES); ?>" />
+                                                        <span><?php echo htmlspecialchars($participant['lname'],ENT_QUOTES) .', ' . htmlspecialchars($participant['fname'],ENT_QUOTES); ?></span>
+                                                    </td>
+                                                    <td><span><?php echo htmlspecialchars($participant['pid'],ENT_QUOTES); ?></span></td>
+                                                    <td>
+                                                        <select name="group_patient_status[]" <?php echo $readonly; ?>>
+                                                            <?php foreach ($statuses as $key => $status): ?>
+                                                                <option value="<?php echo attr($key);?>" <?php if($key == $participant['group_patient_status']) echo 'selected'; ?> > <?php echo xlt($status); ?> </option>
+                                                            <?php endforeach; ?>
+                                                        </select>
+                                                    </td>
+                                                    <td><input type="text" name="group_patient_start[]" id="start-date<?php echo $i+1?>" class="datepicker"  value="<?php echo $participant['group_patient_start'];?>" <?php echo $readonly; ?>></td>
+                                                    <td><input type="text" name="group_patient_end[]" id="end-date<?php echo $i+1?>" class="datepicker" value="<?php echo $participant['group_patient_end'] == '0000-00-00' ? '' : $participant['group_patient_end'] ;?>" <?php echo $readonly; ?>></td>
+                                                    <td><input type="text" name="group_patient_comment[]" class="full-width" class="datepicker"  value="<?php echo $participant['group_patient_comment'];?>" <?php echo $readonly; ?> /></td>
+                                                    <?php if($readonly == ''): ?>
+                                                        <td class="delete_btn">
+                                                            <a href="<?php echo $GLOBALS['rootdir'] . '/therapy_groups/index.php?method=groupParticipants&group_id='. $groupId .'&deleteParticipant=1&pid=' . $participant['pid']; ?>"><span>X</span></a>
```

Comment:
so in href we also need to use escaping methods? Asking because saw many places in code where everything besides for href has escaping